vue/components/ui/atoms/button-color/button-color.vue

```diff
@@ -406,6 +407,11 @@ export const ButtonColor = {
             }
 

```

Comment:
add a `no-text` class for when there's no text and use it to invalidate the padding